### PR TITLE
Added windows update script as update instruction.

### LINKED ISSUE #365: openHAB update script for Windows
I'm not sure if there is an interest in having an upgrade script for Windows or not. There are two files. One is a batch file to call the powershell file as admin and also set the execution policy to bypass.

Batch file "upgrade.bat" Right click, run as administrator.
`
powershell.exe -file %~dp0UpgradeOH2.ps1 -ExecutionPolicy bypass
`

UpgradeOH2.ps1

```powershell
#Requires -RunAsAdministrator

# openHAB upgrade script
#
# Deletes wrapper service
# Renames current install folder for backup
# Unzip new file to existing directory
# Restore \conf folder
# Restore \userdata folder with exclusions
# Restore \runtime folder
# Setup wrapper prompt
# Install wrapper service

######################################## Functions ########################################

Function Get-oHZipFile
{
    [System.Reflection.Assembly]::LoadWithPartialName("System.Windows.Forms") | Out-Null

    $OpenFile = New-Object System.Windows.Forms.OpenFileDialog
    $OpenFile.Title = "Select openHAB .zip file"
    $OpenFile.InitialDirectory = $env:SystemDrive
    $OpenFile.Filter = "ZIP (*.zip)| *.zip"
    $OpenFile.ShowDialog() | Out-Null
    $OpenFile.FileName
}

Function Get-oHDirectory
{
    Add-Type -AssemblyName System.Windows.Forms

    $foldername = New-Object System.Windows.Forms.FolderBrowserDialog
    $foldername.ShowNewFolderButton = $false
    $foldername.RootFolder = "MyComputer"
    $foldername.SelectedPath = $env:SystemDrive
    $foldername.Description = "Select current openHAB directory"

    [void]$foldername.ShowDialog()
    $folder = $foldername.SelectedPath

    return $folder
}

########################################## Code ##########################################

#ask for current install directory
$oHfolder = Get-oHDirectory

#Stop oH service
sc.exe stop "openHAB"

#Delete oH service
sc.exe delete "openHAB"

#Backup oH to new openHABBU folder
$oHParentFolder = $oHfolder.Split('\')[0]
$oHBUFolder = $oHParentFolder + "\openHABBU"
Clear-Host
Write-Host "Backing up current files..."
Copy-Item $oHfolder $oHBUFolder -Container -Recurse
Write-Host "Backed up"

#ask for zip file
Write-Host "Choose .zip file"
Add-Type -AssemblyName PresentationFramework
[System.Windows.MessageBox]::Show('Select openHAB .zip file')
$zipfile = Get-oHZipFile

#Unzip new oH to temp folder openHABNew
Write-Host "Unzipping file to temporary location..."
$oHfolderNew = $oHParentFolder + "\openHABNew"
Expand-Archive $zipfile.ToString() -DestinationPath $oHfolderNew
Write-Host "Unipped"

#Delete current userdata files
Write-Host "Deleting current files that should not persist..."
$userdata = $oHfolder + "\userdata"
Remove-Item ($userdata + '\etc\all.policy')
Remove-Item ($userdata + '\etc\branding.properties')
Remove-Item ($userdata + '\etc\branding-ssh.properties')
Remove-Item ($userdata + '\etc\config.properties')
Remove-Item ($userdata + '\etc\custom.properties')
Remove-Item ($userdata + '\etc\distribution.info')
Remove-Item ($userdata + '\etc\jre.properties')
Remove-Item ($userdata + '\etc\profile.cfg')
Remove-Item ($userdata + '\etc\startup.properties')
Remove-Item ($userdata + '\etc\org.apache.karaf*')
Remove-Item ($userdata + '\cache') -Recurse
Remove-Item ($userdata + '\tmp') -Recurse
Write-Host "Deleted"

#Copy in new userdata files, no overwrite
Write-Host "Copying \userdata\ files to new install..."
Get-ChildItem -Path $oHfolderNew -Recurse | Copy-Item -Destination {
    if ($_.PSIsContainer) {
        Join-Path $oHfolder $_.Parent.FullName.Substring($oHfolderNew.Length)
    } else {
        Join-Path $oHfolder $_.FullName.Substring($oHfolderNew.Length)
    }
} | Out-Null
Write-Host "Copied"

#Copy in new runtime files, overwrite
Write-Host "Copying \runtime\ files to new install..."
Copy-Item -Path ($oHfolderNew + "\runtime\*") -Destination ($oHfolder + "\runtime") -Force
Write-Host "Copied"

#Delete temp new unzipped file
Write-Host "Deleting temp folder containing new files..."
Remove-Item $oHfolderNew -Recurse -Force
Write-Host "Deleted"

#Start openHAB instance
Write-Host "Starting openHAB instance..."
$PI1 = New-Object System.Diagnostics.ProcessStartInfo
$PI1.FileName = $oHfolder + "\runtime\bin\karaf.bat"
$PI1.Arguments = "%*"
$PI1.WorkingDirectory = $oHfolder
$PI1.UseShellExecute = $false
$P1 = [System.Diagnostics.Process]::Start($PI1)

#Ask if installing service
$installService = [System.Windows.MessageBox]::Show("Install the service?", "Install service","YesNo")
switch ($installService) {
    'Yes' {
        Remove-Item ($oHfolder + "\runtime\bin\openHAB-wrapper.exe")
        Remove-Item ($oHfolder + "\userdata\etc\openHAB-wrapper.conf")
        Remove-Item ($oHfolder + "\runtime\bin\openHAB-service.bat")
        Remove-Item ($oHfolder + "\runtime\lib\wrapper\wrapper.dll")
        Remove-Item ($oHfolder + "\runtime\lib\wrapper\karaf-wrapper.jar")
        
        #User interaction to setup wrapper
        [System.Windows.MessageBox]::Show('When openHAB has started, run "openhab:install-service" in console then close this message')
        Write-Host "Run openhab:install-service in console"

        #Install service
        Write-Host "Installing service..."
        $PI2 = New-Object System.Diagnostics.ProcessStartInfo
        $PI2.FileName = $oHfolder + "\runtime\bin\openHAB-service.bat"
        $PI2.Arguments = "install"
        $PI2.UseShellExecute = $false
        $P2 = [System.Diagnostics.Process]::Start($PI2)
        }
    }
```

I tested this in Windows 10. I'm not sure what powershell version is required.

### LINKED ISSUE #996: [windows.md] Windows installation not correct
The windows installation page is incorrect.  I'll be submitting a PR shortly to fix it

### PR DESCRIPTION
**Pull request by Confectrician**
Closes openhab/openhab-distro#917
Closes #996 
Closes #365 

CC #1021 

We should then afterwards talk about a separate section and move this snippet.

@openhab-5iver -> would you like to share your opinion here?

Signed-off-by: Jerome Luckenbach <github@luckenba.ch>

### CODE REVIEW

**Comment by Confectrician** [comment]
Direct preview link: https://deploy-preview-1056--openhab-docs-preview.netlify.com/docs/installation/windows.html

**Comment by openhab-5iver** [comment]
@Confectrician, this looks OK to me, but Windows is not my primary OS.

File: installation/windows.md [hunk 1]
```diff
@@ -72,35 +72,16 @@ Which can be used any time after a failure.
...
-    - `userdata\etc\system.properties`
-    - `userdata\etc\version.properties`
-    - Any file in `userdata\etc` that starts with `org.apache.karaf`
-    - The `userdata\cache` folder
-    - The `userdata\tmp` folder
-    - The `runtime` folder
-
-4.  Copy and paste the contents of the zip file over your existing install, when prompted **do not overwrite existing files**
+To start the update process, run PowerShell as an administrator and use the following commands, while replacing `x.x.x` with the wanted openHAB version.
+Assuming you have openHAB saved in `C:\openHAB`:
```
Comment (openhab-5iver) [inline installation/windows.md]: Assuming you have openHAB installed in `C:\openHAB`:

Comment (openhab-5iver) [reply installation/windows.md]: Actually, the installation instructions suggest to install to C:\openHAB2...

Step 2... https://www.openhab.org/docs/installation/windows.html#installation

File: installation/windows.md [hunk 2]
```diff
@@ -72,35 +72,16 @@ Which can be used any time after a failure.
...
-    - `userdata\etc\system.properties`
-    - `userdata\etc\version.properties`
-    - Any file in `userdata\etc` that starts with `org.apache.karaf`
-    - The `userdata\cache` folder
-    - The `userdata\tmp` folder
-    - The `runtime` folder
-
-4.  Copy and paste the contents of the zip file over your existing install, when prompted **do not overwrite existing files**
+To start the update process, run PowerShell as an administrator and use the following commands, while replacing `x.x.x` with the wanted openHAB version.
+Assuming you have openHAB installed in `C:\openHAB`:
```
Comment (openhab-5iver) [inline installation/windows.md]: C:\openHAB2

File: installation/windows.md [hunk 3]
```diff
@@ -72,35 +72,16 @@ Which can be used any time after a failure.
...
+
+```shell
+cd C:\openHAB
```
Comment (openhab-5iver) [inline installation/windows.md]: C:\openHAB2

**Comment by openhab-5iver** [comment]
Thank you!

**Pull request merged**